src/main/java/org/jabref/gui/ClipBoardManager.java

```diff
@@ -87,6 +87,18 @@ public static String getContents() {
         return result;
     }
 
+    public static String getHtmlContents() {
+        String result = clipboard.getHtml();
+        if (result == null) {
+            return "";
+        }
+        return result;
+    }
+
+    public static boolean hasHtml() {
+        return clipboard.hasHtml();
+    }
```

Comment:
Rub's delegate seems to be syntactic sugar, which is not available in "pure" Java.

There is [Lombok's Delegate](https://projectlombok.org/features/experimental/Delegate), which is kind of similar, but hard to maintain.

In general, the choice of a programming language is not only because of its syntactic features, but more about the eco system - the available libraries and tooling, community, as well as support lifecycle (and overall adaption).

Reply:
Its an additional dependency, not known widely, and it infers with the ByteCode generation. At each Java update, there could be issues making the newer JDK not working. And we want to use the latest JDK - and dont have the resources to contribute to Lombok to fix things there.